### ISSUE TITLE
Swifty Loops

### ISSUE THREAD

**Issue by dasdom** [drkameleon/iswift#1]
In Swift for-loops are written like this:

```
for i in 0..<10 {
    print("i: \(i)")
}
```

[Source](https://developer.apple.com/library/prerelease/ios/documentation/Swift/Conceptual/Swift_Programming_Language/ControlFlow.html#//apple_ref/doc/uid/TP40014097-CH9-ID120)

**Comment by drkameleon** [comment]
Well, this is one of those super-obvious types. Which I missed.

Good job spotting it and thanks for letting me know. (The fix will be included in the very next release).

**Comment by drkameleon** [comment]
Well, that's true. It's just that - as a first step - I decided to be as "loyal" to the original as possible.

However, making it as Swift-style as possible is one of my goals, so what you say is to be included soon. Along with other similar enhancements (e.g. `println` vs `NSLog`)

**Comment by drkameleon** [comment]
Implemented as of the upcoming 1.7.

**Example:**

```objective-c

@implementation aClass

- (void)main {
	for (int i=0; i<10; i++) {
		NSLog(@"i = %d",i);   
	}
    
	for (long k=0; k<=12; k++) {
		NSLog(@"k = %lu", k);
	}
}

@end
```

**Output:**

```swift
class aClass {

	func main() {
		for i in 0..<10 {
			print("i = \(i)")
		}

		for k in 0...12 {
			print("k = \(k)")		
		}
	}

}
```

**Comment by drkameleon** [comment]
Re-opening. Seems broken as of 1.9.

**Comment by drkameleon** [comment]
Fixed (again) as of the upcoming 2.0.
